torch/export/_trace.py

```diff
@@ -1348,6 +1355,196 @@ def _strict_export(
     )
 
 
+def _export_to_aten_ir_make_fx(
```

Comment:
I am trying to make sense of the different of this function and '_export_to_aten_ir` 

There are quite some duplication, and I am not sure about the delta.